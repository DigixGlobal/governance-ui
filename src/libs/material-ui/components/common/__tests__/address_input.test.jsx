import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import AddressInput from '../address_input';

describe('<AddressInput />', () => {
  test('renders correctly', () => {
    const component = shallow(
      <AddressInput />,
    );

    expect(toJson(component)).toMatchSnapshot();
  });
});
