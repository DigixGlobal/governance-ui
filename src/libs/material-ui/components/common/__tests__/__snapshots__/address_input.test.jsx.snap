// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AddressInput /> renders correctly 1`] = `
<span>
  <Input
    action={true}
    className="labeled"
    fluid={true}
    key="0"
    onChange={[Function]}
    placeholder="Enter Address"
    type="text"
    value=""
  >
    <input />
    <Connect(AddressDropdown)
      onChange={[Function]}
      value=""
    />
  </Input>
</span>
`;
